### ISSUE TITLE
LSP: Invalid range specified

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2666]
plugin 19.9.2
Webstorm 2019.2.3

```
java.lang.Throwable at Logger.error : Invalid range specified: (256957, 256956);
Stack trace:
java.lang.Throwable: Invalid range specified: (384, 383); 
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:145)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:252)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:247)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:243)
	at com.intellij.openapi.util.TextRange.<init>(TextRange.java:56)
	at com.intellij.openapi.util.TextRange.<init>(TextRange.java:45)
	at com.google.cloud.tools.intellij.languageclient.LanguageServerExternalAnnotator$Companion.diagnosticToAnnotation(LanguageServerExternalAnnotator.kt:76)
	at com.google.cloud.tools.intellij.languageclient.LanguageServerExternalAnnotator.apply(LanguageServerExternalAnnotator.kt:100)
	at com.google.cloud.tools.intellij.languageclient.LanguageServerExternalAnnotator.apply(LanguageServerExternalAnnotator.kt:53)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doApply(ExternalToolPass.java:216)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$600(ExternalToolPass.java:39)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.lambda$null$0(ExternalToolPass.java:183)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:53)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:936)
```

**Comment by etanshaul** [comment]
@matthewmichihara fyi. this was from 19.9.2, so I'm not sure if it was fixed already

**Comment by matthewmichihara** [comment]
This will be fixed in the upcoming 19.10.1 release.
